### DISCUSSION TITLE
[tool.setuptools] schema

### DISCUSSION THREAD

**Discussion by alexanderankin** [pypa/setuptools#4100] [General]
is there actually a docs page which lists out all the possible fields under this section?

seeing some interesting stuff under here https://github.com/pypa/setuptools/blob/main/setuptools/config/_validate_pyproject/fastjsonschema_validations.py

this is probably it:
https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#setuptools-specific-configuration

**Comment by abravalheri** [comment]
> this is probably it:
https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#setuptools-specific-configuration


That is the page the contains the supported fields.
